sesman/chansrv/sound.c

```diff
@@ -436,6 +469,206 @@ sound_process_output_formats(struct stream *s, int size)
     return 0;
 }
 
+#if defined(XRDP_FDK_AAC)
+
+/*****************************************************************************/
+static int
+sound_wave_compress_fdk_aac(char *data, int data_bytes, int *format_index)
+{
+    int rv;
+    int cdata_bytes;
+    char *cdata;
+
+    AACENC_ERROR error;
+    int aot;
+    int sample_rate;
+    int mode;
+    int bitrate;
+    int afterburner;
+    int channel_order;
+    AACENC_InfoStruct info = { 0 };
+    AACENC_BufDesc in_buf = { 0 };
+    AACENC_BufDesc out_buf = { 0 };
+    AACENC_InArgs in_args = { 0 };
+    AACENC_OutArgs out_args = { 0 };
+    void *in_buffer[1];
```

Comment:
Whats is the point to use an array of single element for the following variables?
I would switch `void **`, `int`